versioned_docs/version-5.x/linter/overview.md

```diff
@@ -12,10 +12,11 @@ ink! includes the linter - a security tool designed to identify typical security
 ## Installation
 The linter is integrated to the contracts build process, therefore you should already have it installed if you are using [`cargo-contract`](https://github.com/paritytech/cargo-contract) of version `4.0.0` or later.
 
-Here is the list of commands you can use for installation:
+Here are commands you can use for installation:
 
 ```
-cargo install cargo-dylint dylint-link ink_linting ink_linting_mandatory
+rustup install nightly-2023-12-28
```

Comment:
Do I need to stick to this nightly toolchain version, or can I also use the stable version? If there are any restrictions, it might make sense to mention them.

Reply:
Yes, you should this exact toolchain, because the linter [depends](https://github.com/paritytech/ink/blob/master/linting/rust-toolchain.toml) on the rustc internal API of that version. I tried the `rustc` version from the corresponding stable release (as far as I recall, `1.75.0`) and it has some changes in the API; therefore, it is not suitable for us.

There are no other restrictions; we should just install that version of the toolchain.

Do you think we should additionally describe to the developers *why* do we need this?

Reply:
I was thinking of adding a note specifying that this is only compliant with this toolchain. It may save some time for someone working with it.

Reply:
Agreed, let's add this.